server/src/models/fake.database.ts

```diff
@@ -1,54 +1,81 @@
 import { User, IUserModel } from './user.model';
 import { Token, ITokenModel } from './token.model';
 import { Post, IPostModel } from './post.model';
+import { Comment, ICommentModel } from './comment.model';
 import { Like, ILikeModel } from './like.model';
 
 import '../helpers/globals';
 import connect from '../connect';
 import { hashPassword } from '../helpers/hash.password';
 
-import mongoose from 'mongoose';
+import mongoose, { Schema, Document, Model, model } from 'mongoose';
 import faker from 'faker/locale/en';
+import axios, { AxiosResponse } from 'axios';
+import FormData from 'form-data';
 
 faker.seed(983465);
 mongoose.set('useNewUrlParser', true);
 
-const { env: { DB_PATH } }: any = process;
+const { env: { DB_PATH, IMAGE_DB_PATH, IMGUR_CLIENT_ID } }: any = process;
 const FAKE_DB_SIZE = parseInt(process.env.FAKE_DB_SIZE, 10);
 
+const capitalizeSentence = function(sentence: string): string {
+    if (!sentence) {
+        return sentence;
+    }
+
+    return sentence[0].toUpperCase() + sentence.slice(1) + '.';
+};
+
 const clearDatabase = async function(path: string): Promise<void> {
     await mongoose.createConnection(path).dropDatabase();
 };
 
 const generateUsers = async function(size: number): Promise<IUserModel[]> {
     const {
-        internet: { email, userName, password },
+        internet: { email, userName, password, avatar },
         name: { firstName, lastName },
         helpers: { shuffle },
         random: { number: randomNumber },
     }: any = faker;
 
-    const users = new Array(size).fill(null).map((throwaway: null) => ({
+    interface ICredentials {
+        email: string;
+        password: string;
+    }
+
+    const credentials: ICredentials[] = new Array(size).fill(null).map((throwaway: null) => ({
         email: email(),
+        password: password(8),
+    }));
+
+    console.log('\nLogin credentials:'); // tslint:disable-line no-console
```

Comment:
console.log?

Reply:
It's CLI tool which builds fake database; console.log, console.table, process.stdout.write are used to plot fake credentials. That won't block main thread in production.